src/pages/AddItemView.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import db from '../lib/firebase';
+import { collection, addDoc } from 'firebase/firestore';
+import AddItemForm from '../components/AddItemForm';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+export const AddItemView = ({ token }) => {
+  const [inputs, setInputs] = useState({
+    days: '7',
+    'last purchase date': null,
+  });
```

Comment:
I recommend saving the days as a number instead of a string. Looking ahead, this value is what you need to calculate the estimate of when it a user will buy an item again.